### DISCUSSION TITLE
Map position

### DISCUSSION THREAD

**Discussion by kurim** [flobz/psa_car_controller#270] [Q&A]
Is it normal that the map position is set to "water" even if we have a position?

**Comment by kurim** [comment]
ok found the root cause -> if you have no data position the map position is set wrong, after delete the database and recapture everything it will be fine.

**Comment by lvenditti** [comment]
Hi,
About the MAP I have a strange behavior:
when the route is recorder the first time on MAP it is displayed almost correctly, but after some days the same route is visualized in wrong position as if the GPS points have been shifted a bit day by day, instead the last recorded route is accurate.

It happen only to me ?

Cheers
Luigi (E-C4)


Last route:
![last_route](https://user-images.githubusercontent.com/57790295/143244481-71886a2f-8402-4857-a2a9-220600661e0e.png)
Last weeks of routes:
![Last_week](https://user-images.githubusercontent.com/57790295/143244510-15e2415f-d4a9-4a6a-a94d-ef83f6c64736.png)

**Reply by grandmastersepp-user** [reply]
Dear Luigi, I see the same phenomenon for my car's data. I will post an issue and link to your finding.

**Reply by kurim** [reply]
yes i can confirm this problem, a route which definitely was right from the city is now left from the city ... 
![image](https://user-images.githubusercontent.com/1880220/146829533-841a6a5e-2a44-40c9-bc89-614b16954ee0.png)

since i never drove on left side of rhein... it is totaly wired.

**Comment by Tinux-Git** [comment]
Hi to all, before upgrade to last version I had the same issue. Now I can't see my old trip. Waiting the issue was solved how do you resolve the issue ?
